README.md

```diff
@@ -1,5 +1,35 @@
 # examples
-Collection of example OpenRPC service definition files
 
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=open-rpc/examples)](https://dependabot.com)
 [![CircleCI](https://circleci.com/gh/open-rpc/examples.svg?style=svg)](https://circleci.com/gh/open-rpc/examples)
+
+This repo contains a collection of example OpenRPC service definition files. These definitions can be found in the `service-desciptions/` directory:
+
+https://github.com/open-rpc/examples/tree/master/service-descriptions
+
+## Getting Started
+
```

Comment:
we can also link to examples via `playground`

https://playground.open-rpc.org?schemaUrl=https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/simple-math-openrpc.json

Reply:
oh hell yeah.